contrib/neovide/template.py

```diff
@@ -0,0 +1,92 @@
+pkgname = "neovide"
+pkgver = "0.12.1"
+pkgrel = 0
+build_wrksrc = "neovide"
```

Comment:
this uses rustls so it should be in user/ repository and have restricted archs set

Reply:
rin builds on ppc64le/riscv64 since 0.17 however (this is 0.17.7). they merged the generic fallback

Reply:
(i.e. this is not restricted because of rustls at all. maybe other reasons..)

Reply:
yeah but it's still statically linked tls so i don't want it in primary repos regardless

Reply:
archs should still be restricted because skia does not support big endian

Reply:
yeah, written as `if self.profile().endian == "big": broken=`

Reply:
no, written as archs= because skia quite literally has arch-specific code in it

Reply:
@q66 what should be the value of `archs =` ?

(I am a complete noob when it comes to architectures)